test/image/mocks/axes_exponentbase_log_e.json

```diff
@@ -0,0 +1,23 @@
+{
+    "data": [{
+        "x": [
+            1,
+            2,
+            3,
+            4
+        ],
+        "y": [
+            1,
+            2.718281828459045,
+            7.3890560989306495,
+            20.085536923187664
+        ],
+        "name": "base e"
```

Comment:
I'm not sure this mock is behaving correctly.

![image](https://cloud.githubusercontent.com/assets/6675409/13056808/b579f990-d3e5-11e5-892f-8b6474b789b9.png)

The minor `1` ticks are really `Math.sqrt(2)` and shouldn't get rounded, right?

I'd even argue that they shouldn't be shown at all.